plugins/woocommerce/client/admin/client/settings-payments/components/enable-gateway-button.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { PaymentGateway } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+
+interface EnableGatewayButtonProps {
+	gateway: PaymentGateway;
+}
+
+const EnableGatewayButton: React.FC< EnableGatewayButtonProps > = ( {
```

Comment:
we could use `<PaymentGatewayButton>` instead

Reply:
Updated in a6b3515157.